rocketchatctl

```diff
@@ -329,7 +360,7 @@ check_arguments_unattended_install() {
 apt_install_node(){
     curl -sL $NODE_DEB_DOWNLOAD_URL | bash -
     apt-get install -y build-essential graphicsmagick
-    apt-get install -y nodejs=8.17.0-1nodesource1
+    apt-get install -y nodejs
```

Comment:
I assume latest is probably the version we want.  But maybe we should still specifically specify the version just in case they release another before we notice

Reply:
I added that before because using n $NODE_VERSION was not working, but later in the npm install function, using n we set to 12.14.0 correctly this time

https://github.com/RocketChat/install.sh/blob/1ab63d215ef5bbfe3a7df84a38f7a8cd23a3bf3b/rocketchatctl#L375

```
# node -v
v12.14.0
```